### PR TITLE
Improve tap docs

### PR DESCRIPTION
**Pull request by Arau**
- Simplify TAP ops steps (and modernise a bit)
- Stop piping cats
- Wrap at 80c according to best practices

### CODE REVIEW

File: docs/operations/tap.md [hunk 1]
```diff
@@ -58,108 +76,93 @@ The following guidance will demonstrate how to use Ondat Topology-Aware Placemen
...
-    ```
+    Check that the primary of the volume and its replicas are placed on
+    different failure domains defined by your custom label.
 
-1. To review and confirm that Ondat TAP has successfully provisioned 3 volumes and evenly distributed in different `custom-region` zones - deploy and run the [Ondat CLI utility as a deployment](https://docs.ondat.io/docs/reference/cli/#run-the-cli-as-a-deployment-in-your-cluster) first, so that you can interact and manage Ondat through `kubectl`. Once deployed, obtain the  Ondat CLI utility pod name for later reference.
+    > 💡 To place 2 replicas, the cluster needs at least `3` nodes (`1` primary
+    >
+    > - `2` replicas). Because of the use of the soft failure-mode, the volume
+    > could operate with 2 nodes while waiting to be able to place a new
+    > replica, eventaully.
```
Comment (cannischan) [inline docs/operations/tap.md:99]: eventually

File: docs/operations/tap.md [hunk 2]
```diff
@@ -58,108 +76,93 @@ The following guidance will demonstrate how to use Ondat Topology-Aware Placemen
...
+        Id:          f0c9a5f8-8034-4524-99cf-0e5602dfd70e
+        Node ID:     defbb2fa-66f5-40b3-86a7-7a167ba2e1ae
+        Promotable:  true
+        Sync Progress:
+      Volume Id:  3d70627d-5122-4255-839b-22f7215393fc
+    ...
     ```
 
-1. With the Ondat CLI now deployed, you can check the location of the master and replica volumes.
+    > 💡As demonstrated in the output above, notice how the primary volume and
```
Comment (cannischan) [inline docs/operations/tap.md:142]: notice -> note?

File: docs/operations/tap.md [hunk 3]
```diff
@@ -172,11 +175,12 @@ The following guidance will demonstrate how to use Ondat Topology-Aware Placemen
     topology.kubernetes.io/zone=northeurope-2
     ```
 
-1. Create a custom `StorageClass`, named `ondat-tap` and check that it has been successfully created.
+1. Create a custom `StorageClass`, named `ondat-tap` and check that it has been
```
Comment (cannischan) [inline docs/operations/tap.md:178]: remove comma

**Review by cannischan — COMMENTED** [review]
small changes needed

**Comment by hubvu** [comment]
@Arau a recommendation would be to remove the $ in the code snippets to make it easier for users to copy the command and run it without mistakenly copying the $ that occurs at the beginning of the snippet.

Overall, LGTM.

**Review by hubvu — APPROVED** [review]

**Review by hubvu — APPROVED** [review]

**Pull request merged**